roles/services/radarr/templates/.env.j2

```
@@ -0,0 +1,15 @@
+# roles/services/radarr/templates/.env.j2
+# Radarr Environment Configuration
+# Generated by Ansible - DO NOT EDIT MANUALLY
+
+# API Configuration
+RADARR_API_KEY={{ lookup('community.general.onepassword', 'radarr', field='api_key', vault=onepassword_vault) }}
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)